notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1614 @@
+{
```

Comment:
<p>Missing link to DiscreteHMCGibbs: it is possible to do so with <code>DiscreteHMCGibbs</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Here is a better link for DiscreteHMCGibbs: https://num.pyro.ai/en/stable/mcmc.html#discretehmcgibbs Typically we link to the docs in num.pyro.ai, rather than the source code.</p>